x-pack/plugins/security_solution/public/management/components/effected_policy_select/utils.ts

```diff
@@ -73,3 +73,12 @@ export function getEffectedPolicySelectionByTags(
 export function isGlobalPolicyEffected(tags?: string[]): boolean {
   return tags !== undefined && tags.find((tag) => tag === GLOBAL_POLICY_TAG) !== undefined;
 }
+
+/**
+ * Given an array of an artifact tags, return the ids of policies inside
+ * those tags. It will only return tags starting with `policy:` and it will
+ * return them without the sufix
+ */
+export function getArtifactPoliciesIdByTag(tags: string[] = []): string[] {
```

Comment:
```suggestion
export function getArtifactPoliciesIdByTag(tags: ImmutableArray<string> = []): string[] {
```

then you won't have to cast the  parameter `as string[]` when you call `getArtifactPoliciesIdByTag`

Reply:
I don't want to pollute helpers with immutable types